docs/observability/aws/deploy-use-aws-observability/before-you-deploy.md

```diff
@@ -181,7 +181,7 @@ Here’s a sample `sources.json` file that you can include in your AMI.
 
 Before setting up the AWS Observability solution we recommend testing permissions for both AWS and Sumo Logic by using a test AWS CloudFormation template. To execute this template:
 
-1. Invoke the AWS CloudFormation template at this [URL](https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.9.0/permissionchecker/permissioncheck.template.yaml).
+1. Invoke the AWS CloudFormation template at this [URL](https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.10.0/permissionchecker/permissioncheck.template.yaml).
```

Comment:
This URL doesn't invoke the CloudFormation template. It only downloads the YAML file. You should get the download URL from the developers.

Reply:
updated URL

Reply:
Thank you!